study.md

```diff
@@ -54,5 +57,6 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+In both, data isn't in a set order
+Enumerables have all the items but streams get items as they are available
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.  Enumerables might not have all of the items available at once (look up [lazy evaluation](https://en.wikipedia.org/wiki/Lazy_evaluation)).